CMakeLists.txt

```diff
@@ -114,11 +134,10 @@ foreach(api ${nidrivers})
       ${nidriver_client_srcs}
       "${service_output_dir}/${api}/${api}_client.cpp")
   endif()
-  GenerateServices(
```

Comment:
Something got messes up with the dependencies in GenerateServices. I think this worked in my original change (which was pretty different ...) but it's possible I missed it. Not sure what the problem is, but obviously we need codegen to work.